website/content/en/docs/config/network/_index.md

```diff
@@ -69,10 +69,27 @@ The configuration steps are different for each network type:
 #### Managed (192.168.105.0/24)
 
 [`socket_vmnet`](https://github.com/lima-vm/socket_vmnet) is required for adding another guest IP that is accessible from the host and other guests.
+It must be installed according to the instruction provided on https://github.com/lima-vm/socket_vmnet.
+
+Note that installation using Homebrew is not secure and not recommended by the Lima project.
+Homebrew installation will only work with Lima if password-less `sudo` is enabled for the current user.
+The `limactl sudoers` command requires that `socket_vmnet` is installed into a secure path only
+writable by `root` and will reject `socket_vmnet` installed by Homebrew into a user-writable location.
 
 ```bash
-# Install socket_vmnet
-brew install socket_vmnet
+# Install socket_vmnet as root from source to /opt/socket_vmnet
+# using instructions on https://github.com/lima-vm/socket_vmnet
+# This assumes that Xcode Command Line Tools are already installed
+git clone https://github.com/lima-vm/socket_vmnet
+cd socket_vmnet
+git checkout v1.1.5
+make
+sudo make PREFIX=/opt/socket_vmnet install.bin
+
+# Alternatively install from the release tarball (use x86_64 instead of arm64 on Intel)
```

Comment:
Maybe use tab panes (See "CLI" and "YAML" tabs in https://lima-vm.io/docs/config/network/)

Reply:
(The binary tar ball should be probably referred in a follow-up PR due to the attestation concern below)

Reply:
I removed the "from release" instructions.